workspaces/theme/plugins/theme/src/utils/createComponents.ts

```diff
@@ -436,7 +436,7 @@ export const createComponents = (themeConfig: ThemeConfig): Components => {
         root: {
           textTransform: 'none',
           minWidth: 'initial !important',
-          '&.Mui-disabled': {
+          '&:disabled': {
             backgroundColor: general.disabledBackground,
```

Comment:
```suggestion
            backgroundColor: 'unset',
```
⬆️ fixes the following issue with tab button background color being gray.
![image](https://github.com/user-attachments/assets/95ca3041-4a27-49b5-9b3c-ce34147b777b)

Reply:
Then I can just remove this disabled code. I thought setting a gray background to the tab was exactly what we want to make it more clear that the tab is disabled.

With just the different text Color is really hard to see if a tab is disabled. I will check with @ShiranHi and the PF 5/6 documentation.

Reply:
@ciiay yeah this "idea" comes from PF. See: https://v5-archive.patternfly.org/components/tabs

![image](https://github.com/user-attachments/assets/3f2f2adf-3914-4492-9be9-3cc71c82a3d2)

Reply:
Right, let's follow [PF guidelines](https://www.patternfly.org/components/tabs/design-guidelines#disabled-tabs). Do they use the same gray color for both light and dark mode?

Reply:
@ShiranHi, I like that you asking the uncomfortable, on-point questions :rofl: 

Here we go: #129